src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6817,13 +6817,114 @@ def test_aks_create_again_should_fail(self, resource_group, resource_group_locat
         self.cmd(cmd, checks=[
             self.is_empty(),
         ])
-    
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
```

Comment:
Seems the feature is not available in westus2?

Reply:
excluded from live-test